CMakeLists.txt

```diff
@@ -47,21 +47,31 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 # Version tokens
+set(NVIM_VERSION_MAJOR 0)
+set(NVIM_VERSION_MINOR 1)
+set(NVIM_VERSION_PATCH 0)
+set(NVIM_VERSION_PRERELEASE "-alpha")
+set(NVIM_VERSION_CODENAME  "Whoa!")
```

Comment:
Dear Michael Reed,

The email sent to danielm@tinyhub.tk with subject
"Re: [neovim] [WIP] CMake: Prepare for releases. (#3318)"
will be delivered as soon as possible.

Your email address is automatically added to the
recipients 'whitelist'. This means for further
emails to danielm@tinyhub.tk, you will not receive a delivery
notification again.

TK MAILIAS
*************************************************
Looking for a way to Protect Your Identity and 
for a great no-more-spam solution for your email
address reply@reply.github.com? Sign up for TK Mailias.

It's free! Please visit http://www.mail.tk/